src/transaction/page_broker.cpp.hpp

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright 2008 Search Solution Corporation
+ * Copyright 2016 CUBRID Corporation
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+#include "page_broker.hpp"  // for convenience - code completion.
```

Comment:
The convention used so far is to include the implementation of inline/template functions in the same header that they're declared.

Please move the content of page_broker.cpp.hpp at the end of page_broker.hpp (a comment may explain where the implementation part starts).